examples/tensorflow-comparison/comparison-training-singlenode.ipynb

```diff
@@ -0,0 +1,242 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "bce78932",
+   "metadata": {},
+   "source": [
+    "# Training using Single GPU\n",
+    "\n",
+    "[Tensorflow](https://rapids.ai/) is a popular, powerful framework for deep learning used by data scientists across industries.\n",
+    "\n",
+    "In this example, you'll train Resnet50 architecture to identify different species of birds. This dataset constitutes 40,000+ birds and has been taken from [kaggle](https://www.kaggle.com/gpiosenka/100-bird-species).\n",
+    "\n",
+    "\n",
+    "First we import necessary libraries."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "323cd78e",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import tensorflow as tf\n",
+    "import keras\n",
+    "import time"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cf4e3b5e",
+   "metadata": {},
+   "source": [
+    "We will be using [Weights & Biases](https://wandb.ai/site) to monitor GPU performance. Users will need to set their own Saturn Cloud env credential for wandb. Check the Saturn Cloud [example on Weights & Biases](https://saturncloud.io/docs/examples/python/weights-and-biases/qs-wandb/) on more information on creating and connecting W&B account to Saturn Cloud. First, we log into Weights & Biases. \n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "448d4326",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import wandb\n",
+    "from wandb.keras import WandbCallback\n",
+    "\n",
+    "wandb.login()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "874cd0fd",
+   "metadata": {},
+   "source": [
+    "### Extracting Data\n",
+    "The dataset originally had 285 classes. We have taken subset of this data which has 61 classes. The data is stored in AWS S3. The first time you run this job, you'll need to download the training and test data which will be saved at path `dataset/birds/`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "bcedb346-9132-4deb-9e07-2ae904943992",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import s3fs\n",
+    "\n",
+    "s3 = s3fs.S3FileSystem(anon=True)\n",
+    "_ = s3.get(\n",
+    "    rpath=\"s3://saturn-public-data/100-bird-species/100-bird-species/*/*/*.jpg\",\n",
+    "    lpath=\"dataset/birds/\",\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "647d38b8",
+   "metadata": {},
+   "source": [
+    "Run the code below to ensure that TensorFlow just uses memory as needed, instead of using all the RAM it has access to on your GPUs."
```

Comment:
```suggestion
    "Run the code below to ensure that TensorFlow uses only the memory required for the model, instead of using all the GPU RAM it has access to."
```